technisch_register/webpages.py

```diff
@@ -93,16 +93,42 @@ def create_overview_entry(standard, title_short, description):
     # print standard
     return BS(overview, 'html.parser')
 
-def create_cluster_overview(standards, source, destination_temp, repoCluster, root, assets_path):
-    """Creates a cluster's overview page
+def create_BRT_overview(standards, source, destination_temp, repoCluster, root, assets_path):
```

Comment:
Is het niet overzichtelijker om de te gebruiken template via een parameter aan de "oude" functie (`create_cluster_overview`) te voeren. Nu ontstaan er twee functies die nagenoeg hetzelfde doen. Of zie ik iets over het hoofd?